### DISCUSSION TITLE
Dreambooth training error : "str2optimizer8bit_blockwise" is not defined

### DISCUSSION THREAD

**Discussion by mahxds** [d8ahazard/sd_dreambooth_extension#934] [Q&A]
Hi , im getting this error while trying to train.

8 bit Adam,fp16, xformers all three checked

anybody know how to fix it ? i cant find anything in google to that error that helps.

Im running it on Ubuntu with a Tesla M40 24GB

**Comment by mwoelke** [comment]
See this issue
https://github.com/TimDettmers/bitsandbytes/issues/62

Specifying the CUDA library path prior to launching the webui worked for me

`export LD_LIBRARY_PATH="CUDAPATH:$LD_LIBRARY_PATH`
where CUDAPATH is your cuda path. For me (on arch system) it was /opt/cuda/targets/x86_64-linux/lib

Don't know exactly where it is on Ubuntu, but maybe /usr/lib/x86_64-linux-gnu/ according to the linked issue.
